content/packages-and-modules/securing-your-code/trusted-publishers.mdx

```diff
@@ -148,6 +148,37 @@ The `id_tokens` configuration tells GitLab to generate an OIDC token for npm. Le
 
 You can modify or remove your trusted publisher configuration at any time through your package settings on [npmjs.com](https://npmjs.com). Each package can only have one trusted publisher connection at a time, but this connection can be edited or deleted as needed. To change providers (for example, switching from GitHub Actions to GitLab CI/CD), simply edit your existing configuration and select the new provider. The change takes effect immediately for future publishes. To remove trusted publishing entirely and return to token-based authentication, delete the trusted publisher configuration from your package settings.
 
+## Recommended: Restrict token access when using trusted publishers
+
+Once you've configured trusted publishers for your package, we strongly recommend restricting traditional token-based publishing access for enhanced security.
+
+### How to configure maximum security
+
+1. After enabling trusted publishers, navigate to your package's **Settings** → **Publishing access**
+2. Select **"Require two-factor authentication and disallow tokens"** 
+3. Save your changes
+
+This configuration:
```

Comment:
I'm not sure if this section is needed, I like the one below better (and think it's enough)

Reply:
I removed it